### PR TITLE
migrate to vercel / next

### PR DESCRIPTION
**Pull request by spencerc99**
weeee 

image

### CODE REVIEW

File: src/api/hello.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```
Comment (jackyzha0) [inline src/api/hello.ts]: remove?

File: src/pages/components/LetterForm.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import * as Dialog from "@radix-ui/react-dialog";
-import { SubmitLetterMetadata } from "../App";
+import { SubmitLetterMetadata } from "../Home";
 import { Letter } from "./Letter";
 
 export function LetterFormButton() {
```
Comment (jackyzha0) [inline src/pages/components/LetterForm.tsx]: Can't seem to comment on the empty file but `src/pages/components/Profile.tsx` would render a `components/Profile` page I think? Could probably keep it under `src/components/Profile`

**Review by jackyzha0 — COMMENTED** [review]
small comments

File: src/pages/components/LetterForm.tsx (hunk 2 above)
Comment (spencerc99) [reply src/pages/components/LetterForm.tsx]: ohhh true, i should probably move all these then huh

**Pull request merged**